src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/mappers/AccessEgressMapper.java

```diff
@@ -15,33 +16,36 @@ public class AccessEgressMapper {
 
   public static List<RoutingAccessEgress> mapNearbyStops(
     Collection<NearbyStop> accessStops,
-    boolean isEgress
+    AccessEgressType accessOrEgress
   ) {
     return accessStops
       .stream()
-      .map(stopAtDistance -> mapNearbyStop(stopAtDistance, isEgress))
+      .map(nearbyStop -> mapNearbyStop(nearbyStop, accessOrEgress))
       .filter(Objects::nonNull)
       .collect(Collectors.toList());
   }
 
   public static Collection<RoutingAccessEgress> mapFlexAccessEgresses(
     Collection<FlexAccessEgress> flexAccessEgresses,
-    boolean isEgress
+    AccessEgressType accessOrEgress
   ) {
     return flexAccessEgresses
       .stream()
-      .map(flexAccessEgress -> new FlexAccessEgressAdapter(flexAccessEgress, isEgress))
+      .map(flexAccessEgress -> new FlexAccessEgressAdapter(flexAccessEgress, accessOrEgress))
       .collect(Collectors.toList());
   }
 
-  private static RoutingAccessEgress mapNearbyStop(NearbyStop nearbyStop, boolean isEgress) {
+  private static RoutingAccessEgress mapNearbyStop(
+    NearbyStop nearbyStop,
+    AccessEgressType accessOrEgress
+  ) {
     if (!(nearbyStop.stop instanceof RegularStop)) {
       return null;
     }
 
     return new DefaultAccessEgress(
       nearbyStop.stop.getIndex(),
-      isEgress ? nearbyStop.state.reverse() : nearbyStop.state
+      accessOrEgress.isEgress() ? nearbyStop.state.reverse() : nearbyStop.state
```

Comment:
uper minor thing, but when there is a natural order to things, it helps the reader to process it in that order:
```
  // This does not follow the natural order
  accessOrEgress.isEgress() ? nearbyStop.state.reverse() : nearbyStop.state
  
  // This do, and is easier to read
  accessOrEgress.isAccess() ? nearbyStop.state : nearbyStop.state.reverse()
```
I will not request you to change this, but keep it in mind for the future - there are many of these in this PR, some was in the wrong order originally some are switched.

Reply:
Yeah, since isEgress was used almost everywhere I got used to that order. But you're right that access is the more obvious default choice.